admin-dev/themes/new-theme/js/pages/product/components/categories/index.js

```diff
@@ -79,16 +83,8 @@ export default class CategoriesManager {
   }
 
   initTree() {
```

Comment:
This will probably become a public method like
`displayTree(array selectedCategoriesIds): void`

That will be called by the manager